src/main/flight/pid.c

```diff
@@ -947,7 +1015,8 @@ void FAST_CODE pidController(const pidProfile_t *pidProfile, timeUs_t currentTim
 #if defined(USE_ACC)
         if ((levelMode == LEVEL_MODE_R && axis == FD_ROLL)
             || (levelMode == LEVEL_MODE_RP && (axis == FD_ROLL || axis ==  FD_PITCH)) ) {
-            currentPidSetpoint = pidLevel(axis, pidProfile, angleTrim, currentPidSetpoint, horizonLevelStrength);
+            rawSetpoint = pidLevel(axis, pidProfile, angleTrim, rawSetpoint, horizonLevelStrength, newRcFrame);
+            currentPidSetpoint = pt3FilterApply(&pidRuntime.attitudeFilter[axis], rawSetpoint);
```

Comment:
The attitude filter could be done earlier, and would also provide the D term filtering, since D term operates on attitude.
I have tested:
```
#define ATTITUDE_CUTOFF_HZ 100 
```